deploy/sre-prometheus/ocm-agent/100-ocm-agent.PrometheusRule.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-managed-notification-alerts
+    role: alert-rules 
+  name: sre-managed-notification-alerts
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-managed-notification-alerts
+    rules:
+    - alert: LoggingVolumeFillingUpNotificationSRE
+    # KubePersistentVolumeFillingUp alert firing in openshift-logging Namespace.
+      expr: ALERTS{alertname="KubePersistentVolumeFillingUp", alertstate="firing", namespace="openshift-logging"} == 1
+      for: 1m
```

As a safe-guard against flapping alerts I think we should not trigger a notification until we know that the alert has been firing consistently for maybe 30m. (arbitrary number, but I feel it should at least be more than 1m)

```suggestion
      for: 30m
```